modules/about/generate_file_stats.php

```diff
@@ -1,29 +1,24 @@
 <?php
 
-$dir = ".";
+$dir = '.';
 
 LetsOpenTheDir($dir);
-    
+
 foreach ($files as $file) {
-    if (preg_match(".php", $file)) {
-        $data = GetTheLinesAndChars($file);
-        $php_lines += $data[0];
-        $php_chars += $data[1];
```

Comment:
The anser is easy: Because I missed that they are counted into separate entries and thought this is just duplicated code. 
To be corrected.

Reply:
Code reverted with prior setup and new regex.
This still rubs me the wrong way, but saw no way to solve this simpler and as readable as the original solution is.
I hate the write-to-php-file after that even more, but I'll rather spend the time elsewhere

Reply:
I would even go one step further: Do we need to have statistics how many PHP files lansuite has?
What benefit does this have?
No user is interested in this (my assumption at least).

From my perspective, the PHP file/line count from the about module can be removed. However, this might be a good thing for another PR.

Reply:
It is kind of nice-to-have and it does show how much work went into this over the years, but I am with you that it would not be the most horrible loss of functionality.
(but also not the most complex to fix and implement properly)
I just wanted to start somewhere, so it was the top of the module list ;)